music21/figuredBass/notation.py

```diff
@@ -410,15 +518,34 @@ class Figure(prebase.ProtoM21Object):
             associated with an expanded
             :attr:`~music21.figuredBass.notation.Notation.notationColumn`.
             ''',
+        'hasExtender': '''
+            A bool value that indicates whether an extender is part of the figure.
+            It is set by a keyword argument.
+            ''',
+        'isExtender': '''
+            A bool value that returns true if an extender is part of the figure but no
+            number is given. Pure extender if you will.
+            It is set by evaluating the number and extender arguments.
+            '''
     }
 
-    def __init__(self, number=1, modifierString=None):
+    isExtender: bool
+
+    def __init__(self, number=1, modifierString=None, extender=False):
         self.number = number
         self.modifierString = modifierString
         self.modifier = Modifier(modifierString)
+        # look for extender's underscore
+        self.hasExtender: bool = extender
+        self._updateIsExtenderProperty()
+        
+    def _updateIsExtenderProperty(self):
+        self.isExtender = (self.number == 1 and self.hasExtender)
 
     def _reprInternal(self):
         mod = repr(self.modifier).replace('music21.figuredBass.notation.', '')
+        if self.isExtender or self.hasExtender:
+            return f'{self.number} {mod} extender: {True}'
```

Comment:
isExtender and hasExtender is redundant, right?  since there should never be a case where something hasExtender but isExtender is False.

Maybe:

```
def _reprInternal(self):
    if self.isPureExtender:
        return 'pure extender'
    mod = repr(self.modifier).replace('music21.figuredBass.notation.', '')
    extender = '___' if self.hasExtender else ''
    return f'{self.number} {mod}{extender}'
```